mix.exs

```diff
@@ -20,7 +20,7 @@ defmodule PhoenixSlim.Mixfile do
   end
 
   def deps do
-    [{:phoenix, "~> 0.16"},
+    [{:phoenix, "~> 1.0.0"},
```

Comment:
I _think_ it should be fine, as long as the API for Phoenix.Template.Engine doesn't change.